cmd/resources/resources.go

```diff
@@ -149,6 +133,42 @@ func GetTemplateData(fileName string) (TemplateData, error) {
 	return TemplateData{Resources: resources}, nil
 }
 
+func NewResourceData(tag openapi3.Tag) ResourceData {
```

Comment:
Made some constructor functions to encapsulate the rules for creating a resource data struct type. This is simpler to test than setting up JSON and calling `GetTemplateData`.